### PR TITLE
[RFR] Fix resolvePermissions not passing value correctly

### LINKED ISSUE #25: 'value' param is always undefined inside func specified in `resolve` prop of WithPermission
Guys. I think you have a little bit confused yourselfs with those funky args naming in `aor-permissions/src/resolvePermissions.js`

Snippet of code from my app:
```
<WithPermission resolve={someFunc} value={'manager'}>
```
```javascript
const someFunc = ({permissions, resource, value, exact, record}) => {
        ...
}
```
**The problem** - the `value` arg is always undefined inside `someFunc` no matter of what.
After spending some time on debugging the transpiled code I've figured out that reason of bug is the following line of code (sorry, I don't have enough time right now for patches so I will provide image :) )

image

After I made the mentioned fix in `node_modules/aor-permissions/lib/resolvePermissions.js` I was able to receive the value I've passed through `value` prop to WithPermission elem

### PR DESCRIPTION
**Pull request by djhi**
Fixes #25

### CODE REVIEW

File: src/resolvePermissions.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 export const resolvePermission = ({ permissions, record, resource }) => mapping => {
     if (typeof mapping.resolve === 'function') {
-        const result = mapping.resolve({ record, resource, permissions, exact: mapping.exact, value: mapping.value });
+        const result = mapping.resolve({
+            record,
+            resource,
+            permissions,
+            exact: mapping.exact,
+            value: mapping.permissions,
```
Comment (ThieryMichel) [inline src/resolvePermissions.js:8]: value can mean anything IMO, is there not a better name ?

**Comment by rmuradov** [comment]
guys, could you issue new `patch release` for `aor-permissions` pckg that would incorporate this merge?
we are really waiting for it cause right now every team member have to fix the bug manually on their local machines in `/node_modules/aor-permissions/lib/resolvePermissions.js`

**Comment by fzaninotto** [comment]
ping @djhi

**Comment by djhi** [comment]
Released in [v1.1.1](https://github.com/marmelab/aor-permissions/releases/tag/v1.1.1)

**Comment by rmuradov** [comment]
thanks!

**Pull request merged**